src/app/modules/index.tsx

```diff
@@ -1,9 +1,7 @@
 import { MainScreenModule } from './MainScreen';
 import { ProfileScreenModule } from './ProfileScreen';
 
-import { NavigationStructureTypes } from '../shared/types/navigation';
-
-export const navigationStructure: NavigationStructureTypes = {
+export const navigationStructure = {
```

Comment:
let's have something like that as well:

```
interface NavigationStructureProps {
  [key: string]: {
    screen: NavigationScreenComponent<any, any>;
    navigationOptions: NavigationStackOptions;
  };
}

export const navigationStructure: NavigationStructureProps =  { ...}

```

While `NavigationStructureProps` would be placed in `namespace` folder